allennlp/service/server_flask.py

```diff
@@ -154,6 +154,12 @@ def predict(model_name: str) -> Response:  # pylint: disable=unused-variable
         if request.method == "OPTIONS":
             return Response(response="", status=200)
 
+        # Do log if no argument is specified
+        record_flag = not request.args.get("record", "true").lower() == "false"
```

Comment:
`!=` would be more idiomatic (and more readable) than `not ==`

Reply:
also, I would prefer more descriptive names than `_flag`, e.g. `record_to_database` and `check_cache`